java/src/client/communication/ChatController.java

```diff
@@ -22,4 +27,11 @@ public void sendMessage(String message) {
 
     }
 
+    @Override
+    public void update(Observable o, Object arg) {
+        if (o instanceof Populator && arg instanceof ModelFacade) {
```

Comment:
What would happen here, is you would just go to your Populator/ModelFacade pointer and get the information from there.